### ISSUE TITLE
The exported mesh cannot be restored after being loaded using nvdiffmodeling

### ISSUE THREAD

**Issue by Lucklycat** [NVlabs/nvdiffmodeling#29]
Thank you very much for your work！
Taking ewer as an example, I re-use nvdifmodeling to load and render the exported mesh. The code is as follows:
`
    eye = [0.5, 0.0, 3.0]
    up = np.array([0.0, 1.0, 0.0])
    at = np.array([0, 0, 0])
    a_mv = util.lookAt(eye, at, up)
    a_mvp = np.matmul(proj_mtx, a_mv).astype(np.float32)[None, ...]
    a_lightpos = np.linalg.inv(a_mv)[None, :3, 3]
    a_campos = np.linalg.inv(a_mv)[None, :3, 3]

    background = torch.tensor(util.checkerboard(2048, 8), dtype=torch.float32, device='cuda')

    params = {'mvp': a_mvp, 'lightpos': a_lightpos, 'campos': a_campos,
              'resolution': [2048, 2048],
              'time': 0}

    _opt_ref = mesh.center_by_reference(render_ref_mesh.eval(params), ref_mesh_aabb, 2.0)
    img_ref = render.render_mesh(glctx, _opt_ref, a_mvp, a_campos, a_lightpos, 20.0, 2048,
                                 num_layers=1, spp=1, background=background,
                                 min_roughness=0.08)

    img_ref = util.scale_img_nhwc(img_ref, [2048, 2048])
    result_image = util.tonemap_srgb(img_ref[0])
    np_result_image = result_image.detach().cpu().numpy()
    util.save_image(out_dir + '/' + ('test_%d.png' % img_cnt), np_result_image)
    img_cnt = img_cnt + 1
`
It ensures that all parameters are consistent, but it is found that the rendered image result after import is far from the final result during training:
![image](https://user-images.githubusercontent.com/17697877/217476563-bb00210c-5767-4ac1-801f-0a8a0faea1a2.png)

Using the same code, I imported the original ewer's obj, and the rendered image is as follows, which proves that there is no problem with the super-parameter setting：
![image](https://user-images.githubusercontent.com/17697877/217476990-6ade221b-8961-466c-bb92-f2f7447bc36d.png)

What is the reason for this problem?

**Comment by jmunkberg** [comment]
In the first image above, it looks like the left example is rendered without the normal map

**Comment by Lucklycat** [comment]
> In the first image above, it looks like the left example is rendered without the normal map

Thank you for your reply @jmunkberg . 
I use obj.write directly_ For the obj generated by obj, I checked the matching. mtl, which contains the normal map:
![image](https://user-images.githubusercontent.com/17697877/217481352-b6dfc90c-3f15-46c5-8e03-3b57e84400e5.png)

If it didn't take effect because normal map was not imported, I debugged the import code. From the perspective of structure, it should be normal map:
![image](https://user-images.githubusercontent.com/17697877/217483266-29ca4ca6-1e17-4b6d-95e2-5e04de1d83bd.png)

However, from lines 45 to 68, we can see that to include 'normal' in mat, '' must exist before it can be imported. However, from the generation of the. mtl file, it does not contain '', so whether there is a problem with the code here，In this way, it is difficult to import the normal map
![image](https://user-images.githubusercontent.com/17697877/217485134-2387ebfa-b7d7-4ca7-8b3e-623846e78627.png)



The import code and pre-processing code are as follows:
``` 
proj_mtx = util.projection(x=0.5, f=1000.0)
ref_mesh = load_mesh(path, None)
render_ref_mesh = mesh.compute_tangents(ref_mesh)
ref_mesh_aabb = mesh.aabb(render_ref_mesh.eval())
 glctx = dr.RasterizeGLContext()
```
